Gruntfile.js

```diff
@@ -494,16 +513,19 @@ module.exports = function(grunt) {
         ]
     });
 
+    grunt.loadNpmTasks('grunt-w3c-html-validation');
+
     grunt.registerTask('default', [
```

Comment:
How long does the validation task take? Is it something we want to run on the default task, or something to roll into a testing/pre-release task?